docs/governance/epochs/epoch1.md

```diff
@@ -0,0 +1,37 @@
+- **Start**: Feb 1, 2022
+- **HIP cutoff**: May 31, 2022
+- **End, PRP cutoff**: June 30, 2022
+- **Goal**: Distribute 30 million HBOT tokens
+
+The last day to submit [Hummingbot Improvement Proposals (HIPs)](./proposals/hip) requesting a share of Dev Grant Budgets is May 31, 2022, and the last day to submit [Pull Request Proposals (PRPs)](./proposals/prp) linked to approved HIPs is June 30, 2022.
+
+| Category                                                                      | Total HBOT Available      | Target Number of Grants |
+| ------------------------------------------------------------------------------| ------------------------- | ----------------------- |
+| [Liquidity Distributions](/governance/proposals/hgp/#liquidity-distributions-ld) | 15,000,000                | 10 platforms             |
+| [Dev Grant Budget](/governance/proposals/hgp/#dev-grant-budget-dg)               | 10,000,000                | 50 dev grants           |
+| [Initiative Budget](/governance/proposals/hgp/#initiative-budget-ib)     | 5,000,000                 | 5 initiatives           |
+
+### HBOT Guidance
+
+In requesting HBOT grants for Hummingbot Improvement Proposals (HIPs) related to a community-approved [Dev Grant Budget](/governance/proposals/hgp/#dev-grant-budget-dg), developers may need guidance in determining how much HBOT to request for unit of time/effort spent on a given task.
+
+For Epoch 1, the Hummingbot Foundation believes that **20,000 HBOT for one day of work** is a reasonable amount for an experienced developer to request for a valid HIP. However, developers are free to request more or less than this amount in their proposal.
+
+## Epoch 1 Budget allocation
+
+### HBOT allocation
+|Approved Proposals                                  |   Amount       |Voting Status      |
+|----------------------------------------------------|----------------|-------------------|
+|[HGP-2](https://snapshot.org/#/hbot.eth/proposal/0x3971d8250ba82cd631141949dbfc77f2df11e99547b265074cab3155c8c36ad0)| 5,000,000 | Approved |
+|[HGP-4](https://snapshot.org/#/hbot.eth/proposal/0xd0c5b54badfd631d7433da0f76795a9dc0d82fc66596d547cda2f3537f903e3f)| 1,000,000 | Approved |
+|[HGP-5](https://snapshot.org/#/hbot.eth/proposal/0xc2b0005a70183758d9bec46a0b532ddebb3e903738e4ba17302c5dd66ec874a0)| 2,000,000 | Voting Open |
```

Comment:
Remove the Voting Open proposals. We don't want to set the expectation that we will keep updating this doc.

Reply:
done